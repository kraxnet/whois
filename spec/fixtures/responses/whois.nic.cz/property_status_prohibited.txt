%  (c) 2006-2011 CZ.NIC, z.s.p.o.
%
% Intended use of supplied data and information
%
% Data contained in the domain name register, as well as information
% supplied through public information services of CZ.NIC association,
% are appointed only for purposes connected with Internet network
% administration and operation, or for the purpose of legal or other
% similar proceedings, in process as regards a matter connected
% particularly with holding and using a concrete domain name.
%
% Full text available at:
% http://www.nic.cz/page/306/intended-use-of-supplied-data-and-information/
%
% See also a search service at http://www.nic.cz/whois/
% 
% 
% Whoisd Server Version: 3.4.0
% Timestamp: Sat May 26 12:22:45 2012

domain:       tollmi.cz
registrant:   SB:TOL453-20020311S
admin-c:      TOL453-20020311P
nsset:        NSS:CZ.VOL:2
registrar:    REG-DIAL-TEL
status:       Delete prohibited
status:       Registration renewal prohibited
status:       Update prohibited
status:       Domain is administratively kept in zone
status:       Domain blocked
registered:   11.03.2002 18:16:00
changed:      16.03.2006 15:45:00
expire:       10.03.2009

contact:      SB:TOL453-20020311S
org:          TOLLMI Milevsko spol. s r.o.
name:         TOLLMI Milevsko spol. s r.o.
address:      Hrejkovice 1
address:      Hrejkovice
address:      39901
address:      CZ
e-mail:       tollmi@volny.cz
registrar:    REG-DIAL-TEL
created:      11.03.2002 18:16:00
changed:      31.07.2008 10:04:23

contact:      TOL453-20020311P
name:         Jindrich Tollinger
address:      Hrejkovice 1
address:      Hrejkovice
address:      39859
address:      CZ
phone:        +420 382525046
fax-no:       +420 382522914
e-mail:       tollmi@volny.cz
registrar:    REG-DIAL-TEL
created:      11.03.2002 18:16:00

nsset:        NSS:CZ.VOL:2
nserver:      names.dialtelecom.cz
nserver:      sns.dialtelecom.cz
tech-c:       RYSLINKD_COL
tech-c:       SB:DIAL-T
registrar:    REG-DIAL-TEL
created:      01.10.2007 02:00:00
changed:      16.05.2011 16:07:32

contact:      RYSLINKD_COL
org:          Dial Telecom, a.s.
name:         Daniel Ryslink
address:      Křižíkova 36a/237
address:      Praha 8
address:      186 00
address:      CZ
phone:        +420.226204400
fax-no:       +420.246000118
e-mail:       daniel.ryslink@dialtelecom.cz
registrar:    REG-DIAL-TEL
created:      02.01.2002 13:21:00
changed:      16.11.2011 13:33:20

contact:      SB:DIAL-T
org:          Dial Telecom, a.s.
name:         Dial Telecom, a.s.
address:      Krizikova 36a/237
address:      Praha 8
address:      186 00
address:      CZ
e-mail:       admin@dialtelecom.cz
registrar:    REG-GENREG
created:      19.10.2006 22:35:00
changed:      06.12.2007 15:27:01
