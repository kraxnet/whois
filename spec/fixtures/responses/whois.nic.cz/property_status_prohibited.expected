#status
  should: %s == [:server_delete_prohibited, :server_renew_prohibited, :server_update_prohibited, :server_in_zone_manual, :server_blocked]

#available?
  should: %s == false

#registered?
  should: %s == true
